build-scripts/build-redisai-cpu.sh

```diff
@@ -15,7 +18,7 @@ else
     echo "Downloading RedisAI CPU dependencies"
     CC=gcc CXX=g++ WITH_PT=1 WITH_TF=1 WITH_TFLITE=0 WITH_ORT=0 bash get_deps.sh cpu
     echo "Building RedisAI"
-    CC=gcc CXX=g++ GPU=0 WITH_PT=1 WITH_TF=1 WITH_TFLITE=0 WITH_ORT=0 WITH_UNIT_TESTS=0 make -C opt clean build
+    CC=gcc CXX=g++ GPU=0 WITH_PT=1 WITH_TF=1 WITH_TFLITE=0 WITH_ORT=0 WITH_UNIT_TESTS=0 make -j $NPROC -C opt clean build
```

Comment:
I believe we can just use ``$(nproc)`` here as some SmartRedis users might not be using Python. Right?

Speaking of this.. are we requiring Python in order to build because of the python based cmake? Can a user build without Python? Should they?